mods/cnc/rules/defaults.yaml

```diff
@@ -480,10 +480,13 @@
 		SellSounds: cashturn.aud
 	Capturable:
 	WithMakeAnimation:
+		MakeUpgrades: underConstruction
+	DisableOnUpgrade:
+		UpgradeTypes: underConstruction
```

Comment:
I think it's much worse when it involves adding yet another inconsistency to already horribly inconsistent files.